src/panels/lovelace/common/entity/compute-lovelace-entity-name.ts

```diff
@@ -21,7 +21,7 @@ export const computeLovelaceEntityName = (
   if (!config) {
     return stateObj ? computeStateName(stateObj) : "";
   }
-  if (typeof config === "string") {
+  if (typeof config === "string" || typeof config === "number") {
     return config;
   }
```

Comment:
Let's make this more defensive

```suggestion
  if (typeof config !== "object") {
    return String(config);
  }
```